README.md

```diff
@@ -16,42 +15,33 @@ Please see the [GitHub template repository documentation](https://help.github.co
 Requirements
 ------------
 
--	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
--	[Go](https://golang.org/doc/install) 1.11
+-	[Terraform](https://www.terraform.io/downloads.html) 0.12.x
+-	[Go](https://golang.org/doc/install) 1.12
```

Comment:
we could recommend 1.14

Reply:
I guess TF 0.12 is built on 1.12... not sure if there is any real risk of a 1.12 built talking to a 1.14 built provider

Reply:
1.12 isn't even listed anymore, so yeah we should just bump to 1.14

Reply:
https://golang.org/dl/

Reply:
We could recommend 1.14 as it's the current Go version, but it's not _required_ for provider development. Provider developers may be working in environments where they are unable to upgrade the Go version. From the SDK point of view, Go 1.12+ is required.